template.adoc

```diff
@@ -0,0 +1,17 @@
+= [short title describing spike]
+Satoshi Nakamoto <satoshi.nakamoto@coblox.tech>;
+:toc:
```

Comment:
As mentioned in the other document, I think having the ToC right at the front looks a bit off. I'd prefer having it after the meta-data (just before the content starts).

Reply:
No we can't as far as I understand/tried. `:toc:` keyword needs to be just after the title.

Reply:
Just checked the documentation it is possible but complicated :roll_eyes: 

http://asciidoc.org/userguide.html

This mentions the `toc-placement` attribute, which can be set to `manual` which allows you to use the `toc::[]` block macro to place the table of contents where you want.

Reply:
As you saw on my laptop, neither CLion asciidoc plugin nor GitHub asciidoc rendering supports this `toc::[]` macro....

Reply:
Sad that it doesn't work :(